src/server/models/user.ts

```diff
@@ -31,6 +28,12 @@ export const UserSchema = new Schema<User, Model<User, User>>({
     type: String,
     required: true,
   },
+  role: {
+    type: String,
+    required: true,
+    enum: ['admin', 'manager', 'user'],
+    default: Role.User,
```

Comment:
Having a `default` role as user will create users with `User` role when no role is explicitly specified, thus having an unexpected result.

The `role` property must remain `required`, and should always be specified.